### DISCUSSION TITLE
[NIGNX-LUA] Is there any way to set SkyWalking token authentication?

### DISCUSSION THREAD

**Discussion by wywanlv** [apache/skywalking#9644] [Q&A] [answered]
I am integrating SkyWalking into nginx with nginx-lua. Is there any way to set token authentication config which SkyWalking server required?

**Comment by wu-sheng** [comment, marked as answer]
I think we don't support that for now. But this is easy to add, welcome to submit your pull request.

**Reply by wywanlv** [reply]
thanks a lot for replying
